templates/components/alert.html.twig

```diff
@@ -38,10 +38,18 @@
             {% if _title is not empty or _description is not empty %}
                 <div>
                     {% if _title is not empty %}
-                        <h3 class="{{ _description is not empty ? 'mb-1' : 'mb-0' }}">{{ _raw ? _title|raw : _title }}</h3>
+                        {% if _background %}
+                            <h3 class="{{ _description is not empty ? 'mb-1' : 'mb-0' }}">{{ _raw ? _title|raw : _title }}</h3>
```

Comment:
why is it getting bigger with background?

Reply:
Because Tabler doesn't handle title/description content as the sames way if `alert-important` or not.

#### normal : h4 + div
![image](https://user-images.githubusercontent.com/25293190/171813862-f369ab0f-9b05-4ac7-a84d-39cc4c3d7f22.png)

#### `alert-important` : d-flex + divs
![image](https://user-images.githubusercontent.com/25293190/171813985-6b38b561-015e-4320-8724-4d6fbe68cd26.png)

Reply:
Hm, annoying that we cannot re-use the same structure. 
Then there are 8 flows:
- important, title, title + description, description
- not important, title, title + description, description
and each has a different structure?

Reply:
As soon Tabler uses icon or other (in column) with alerts, they add `d-flex` container.
![image](https://user-images.githubusercontent.com/25293190/171848701-6cad29bf-7c52-44de-8843-92811a3f8d82.png)

That why I've choose to "getting bigger with background" instant of a new whole block.